live-site/lib/types.ts

```diff
@@ -32,12 +32,32 @@ export interface EventScheduleTabProps {
 export interface StyledTabTitleWrapperProps {
   isSelected: boolean;
 }
+export interface EventItemTagProps{
+  type:string
```

Comment:
nit can we change this to tagType? type is kinda vague and we dont want to confuse it with js types like number, string, etc